resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -792,6 +838,40 @@ public class ApiClient implements AutoCloseable {
         }
     }
 
+    public static class RetryConfigBuilder {
```

Comment:
Having a builder for 3 properties isn't necessary, but if you want to keep it, it should be created using the same pattern as the API client's builder with a private constructor that is only used by the `standard()` static method.